ee/codegen/src/__test__/__snapshots__/generate-code.test.ts.snap

```diff
@@ -37,3 +37,35 @@ class UseApiWithSecret(FinalOutputNode[BaseState, Any]):
         value = ApiWithSecret.Outputs.json
 "
 `;
+
+exports[`generateCode > should generate code for %1 code-execution-node-with-await-all.ts > nodes/code_execution_with_await_all/__init__.py 1`] = `
+"from vellum.workflows.nodes.displayable import CodeExecutionNode
+from vellum.workflows.state import BaseState
+
+
+class CodeExecutionWithAwaitAll(CodeExecutionNode[BaseState, str]):
+    filepath = "./script.py"
+    code_inputs = {}
+    runtime = "PYTHON_3_11_6"
+    packages = None
+"
```

Comment:
Ok now that we are able to reproduce the bug, update our `vellum` serializers so that we properly codegen the trigger class here